src/lib.rs

```diff
@@ -85,7 +86,7 @@ impl<V> IntMap<V> {
 
     /// Ensures that the IntMap has space for at least `additional` more elements
     pub fn reserve(&mut self, additional: usize) {
-        let capacity = (self.count + additional).next_power_of_two();
```

Comment:
Was `.next_power_of_two()` necessary here? `self.lim()` always returned a power of two, so the desired property should already hold by checking `self.lim() < self.count + additional`.

Reply:
I think it was left over from a older implementation where I actually used `capacity`, should be redundant now.